assets/queries/k8s/audit_policy_not_cover_key_security_concerns/query.rego

```diff
@@ -0,0 +1,41 @@
+package Cx
+
+import data.generic.common as common_lib
+import data.generic.k8s as k8sLib
+import future.keywords.in
+
+resources := {
+	{"resource": "secrets","levels": ["Metadata"]},
+    {"resource": "tokenreviews","levels": ["Metadata"]},
+    {"resource": "configmaps","levels": ["Metadata"]},
+    {"resource": "pods","levels": ["Metadata"]},
+    {"resource": "deployments","levels": ["Metadata"]},
+    {"resource": "pods/exec","levels": ["Metadata","Request","RequestResponse"]},
+    {"resource": "pods/portforward","levels": ["Metadata","Request","RequestResponse"]},
+    {"resource": "pods/proxy","levels": ["Metadata","Request","RequestResponse"]},
+    {"resource": "services/proxy","levels": ["Metadata","Request","RequestResponse"]}
+}
+
+CxPolicy[result] {
+    resource:= input.document[i]
+    resource.kind == "Policy"
```

Comment:
Please, add `Policy` in line 31, in the file `kics/.github/workflows/validate-k8s-samples.yml`

![image](https://user-images.githubusercontent.com/74001161/160581352-0951373e-eb1f-4cac-810a-1bd11416ae47.png)